### ISSUE TITLE
creating poll not possible as blind user

### ISSUE THREAD

**Issue by guenterhanke** [kellerben/dudle#142]
As a blind creating a poll is not possible.

I'm using Windows 10, Chrome and the opensource screenreader NVDA.

I can click on create and can enter a title but cannot manage to do the following things:

- skip to the next or previous month
there is simply no link or graphic that lets me do this.

- I cannot manage to add a date
- I go to the date and press enter, the page refreshes but i cannot see that the date was added.
- 
- - in some dates my screenreader tells me
- Wiedergabe
- wich is the german localisation for "Play".
- But what should be played here?
In the end i'm not able to add dates to the list and therefore cannot finish the creation process.

This is really a pitty!
And against German and European law!

See the WCAG from W3C!

**Comment by bkmgit** [comment]
Thanks for reporting this. https://wave.webaim.org indicates that:
- Icons to change the month are missing text
- the language is missing from the html header

**Comment by bkmgit** [comment]
When using this in the browser, the tab key allows one to select different elements on the page to create an event poll. Are the date and month options picked up by NVDA?
